index.html

```diff
@@ -2,9 +2,137 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <link rel="stylesheet" href="styles.css">
+    <title> Rekupe Website</title>
 </head>
 <body>
+<header>
+    <nav class="navigation-bar">
+        <ul class="navigation-list">
+            <li><a href="#case-studies" class="jump-link-nav"> Case Studies</a></li>
+            <li><a href="#services" class="jump-link-nav"> Services</a></li>
+            <li><a href="#contact-us" class="jump-link-nav"> Contact Us</a></li>
+        </ul>
+    </nav>
 
+    <div class="introduction">
+        <h1 class="introduction-header">Your website is an investment in the success of your business.</h1>
+        <p class="introduction-line">We are here to help you succeed.</p>
+        <button type="button" class="work-together-button"> Let's work together </button>
+    </div>
+        <img class= "introduction-image" src="assets/squiggle_line_stripe.svg" alt="squiggle_line_stripe">
+
+
+</header>
+
+<main>
+    <section class="hello">
+        <div class=work-sections>
+            <h2 class="hello-title">Rekupe is a digital agency in Los Angeles.</h2>
+            <h2 class="hello-title">We use strategic UX, SEO optimization and beautiful visual designs to help your
+                business grow.</h2>
+            <h2 class="hello-title">We see every project as a fresh opportunity, pushing ourselves to create new digital
+                experiences.</h2>
+        </div>
+        <p class="question-link">Have a question? <a href="#contact-us" class="contact-us-link">Click here</a> to
+            contact us!</p>
+    </section>
+
+    <section id="case-studies" class="work-part-one">
+        <div class=work-sections>
+            <h3> Rolling Stone</h3>
+            <img src="assets/squiggle_line.svg" alt="squiggle-line" class="squiggle-line">
+            <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+            <a href="http://"> view project</a>
+        </div>
+        <img src="assets/work-image-01.jpeg" alt="work-image-01" class="work-images">
```

Comment:
Wanneer je het venster smaller maakt schalen de plaatjes niet netjes. In de omschrijving staat o.a. dat het ook in een span gezet moet worden